android/app/src/main/java/im/status/ethereum/MainApplication.java

```diff
@@ -52,5 +52,6 @@ public ReactNativeHost getReactNativeHost() {
     public void onCreate() {
         super.onCreate();
         SoLoader.init(this, /* native exopackage */ false);
+        WebView.setWebContentsDebuggingEnabled(true);
```

Comment:
why `BuildConfig.DEBUG_WEBVIEW == "1")` isn't used? don't think we want this enabled in release

Reply:
Oh, sorry, did not know about the flag. Will fix.